apps/challenges/views.py

```diff
@@ -2672,7 +2672,7 @@ def manage_worker(request, challenge_pk, action):
     return Response(response_data, status=status.HTTP_200_OK)
 
 
-@api_view(["GET"])
+@api_view(["POST"])
```

Comment:
Why are we changing it to `POST` request?

Reply:
@RishabhJain2018 because we added these request data params https://github.com/Cloud-CV/EvalAI/pull/3057/files#diff-3ce6744a64dac7bb4f8172e9ceeb1a3b2717399cbeec67483b1ac98574601eb8R2706.